README.md

```diff
@@ -25,8 +25,9 @@ please [submit an issue][issues] or [mail us][email].
     `YYYY-MM-DD` is the start date of the workshop, `ds` stands for Digital
     Skills, and `gpu` is the workshop name in this example.
 
-4.  Make sure the repository is public, leave "Include all branches" unchecked,
-    and click on "Create repository from template". You will be redirected to
+4.  Make sure the repository is public, leave "Include all branches" unchecked 
+    (don't worry about the `gh-pages` branch, it will be created by a github action).
```

Comment:
Aha! That I did not realize when I went through this workflow. 
Perhaps it is good then to add a step either between 4 and 5, or as part of 5, to verify that the actions have completed, and that the gh-pages branch is created? Or warn the user that it does take a little while; when I tried it out just now this Action took 5 minutes to complete.

Reply:
Yes, that is important info.

Reply:
Added.